packages/graphql-hooks/package.json

```diff
@@ -35,6 +35,7 @@
   },
   "dependencies": {
     "dequal": "^2.0.0",
+    "events": "^3.3.0",
```

Comment:
Will this new dependency have an impact on the bundle size?

Reply:
![Screenshot 2021-11-18 at 12 00 21](https://user-images.githubusercontent.com/630796/142403252-6e83728e-21db-4fcb-8148-c1a531c5fc3d.png)

Minimal, but there is. Do you believe that we are still targeting the goal of this project of having a minimal bundle size?

Reply:
looks good

Reply:
It does seem to be a selling point as far as I can see but quite new to the library/project so I may not be the most qualified person to say :-) I've seen people using a js map to track subscribers (I think it was in xState but I can't find it now.). Maybe this could be used if size is an issue.

Reply:
there isn't a strong argument in favor of this library but overall it looks like a reasonable compromise